plopfile.js

```diff
@@ -82,4 +65,37 @@ module.exports = function(plop) {
       ];
     },
   });
+
+  plop.setGenerator('tsconfig', {
```

Comment:
probably we should change this and the docs command to be "custom actions" and then have the new package generator consume them